BookingsAPI/Bookings.AcceptanceTests/Steps/SuitabilityAnswersSteps.cs

```diff
@@ -0,0 +1,94 @@
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using TechTalk.SpecFlow;
+using Testing.Common.Builders.Api;
+
+namespace Bookings.AcceptanceTests.Steps
+{
+    [Binding]
+    public sealed class SuitabilityAnswersSteps
+    {
+        private readonly TestContext _acTestContext;
+        private readonly ParticipantsEndpoints _endpoints = new ApiUriFactory().ParticipantsEndpoints;
+        private readonly SuitabilityAnswerEndpoints _suitabilityEndpoints = new ApiUriFactory().SuitabilityAnswerEndpoints;
+        private readonly HearingsEndpoints _hearingEndpoints = new ApiUriFactory().HearingsEndpoints;
+
+        public SuitabilityAnswersSteps(TestContext acTestContext)
+        {
+            _acTestContext = acTestContext;
+        }
+
+        [Given(@"I have the suitable answers submitted")]
+        public void GivenIHaveTheSuitableAnswersForParticipants()
+        {
+            AddSuitabilityAnswersToParticipants(_acTestContext.HearingId, _acTestContext.Participants);
+        }
+
+        [Given(@"I have a request to get the suitable answers")]
+        public void GivenIHaveARequestToGetTheSuitableAnswers()
+        {
+            _acTestContext.Request = _acTestContext.Get(_suitabilityEndpoints.GetSuitabilityAnswers(""));
+        }
+
+        [Given(@"I have a request to the second set of suitable answers")]
+        public void GivenIHaveARequestToTheSecondSetOfSuitableAnswers()
+        {
+            //Create hearing and submit answers to participants
+            AddSuitabilityAnswersToHearing();
+            AddSuitabilityAnswersToHearing();
+            
+            //Get first set of the suitability answers
+            var suitabilityGetRequest = _acTestContext.Get(_suitabilityEndpoints.GetSuitabilityAnswerWithLimit("", 1));
+            var response = _acTestContext.Client().Execute(suitabilityGetRequest);
+            var firstSetOfRecords = ApiRequestHelper.DeserialiseSnakeCaseJsonToResponse<SuitabilityAnswersResponse>(response.Content);
+
+            //Get 2nd set of the suitability answers
+            _acTestContext.Request = _acTestContext.Get(_suitabilityEndpoints.GetSuitabilityAnswerWithLimit(firstSetOfRecords.NextCursor, 1));
+        }
+
+
+        [Then(@"suitable answers should be retrieved")]
+        public void ThenSuitableAnswersShouldBeRetrieved()
+        {
+            var model = ApiRequestHelper.DeserialiseSnakeCaseJsonToResponse<SuitabilityAnswersResponse>(_acTestContext.Json);
+            model.Should().NotBeNull();
+            model.PrevPageUrl.Should().NotBe(model.NextPageUrl);
+            model.ParticipantSuitabilityAnswerResponse.Should().NotBeNull();
+            model.ParticipantSuitabilityAnswerResponse.Count.Should().BeGreaterThan(0);
+            var participantAnswer = model.ParticipantSuitabilityAnswerResponse[0];
+            participantAnswer.FirstName.Should().NotBeEmpty();
```

Comment:
we probably should compare the data based on what we added previously. for example, firstname and lastname could be swapped here without being caught.

Reply:
I don't even think we should check for the first and last data here, as it would have been covered in a different scenarios and tests. The main purpose of the test is to verify suitability  answers. What do you think ?

Reply:
It would've been covered in unit tests true, I reckoned the acceptance tests would do at least a happy test of one set of data though? But maybe it's fine to have a green passthrough, as a smoke test. @npoxon ?

Reply:
I didn't mean unit tests, I meant other acceptance that were specifically written verify these data such as AddParticipant etc in which case we should be verifying the names.

Reply:
In which case this data isn't being caught, it's a different endpoint with different data collection and mapping so I think we ought to test it in that case